docs/product/components/inputs.html

```diff
@@ -333,3 +333,64 @@
         </div>
     </div>
 </section>
+
+<section class="stacks-section">
+    {% header "h2", "Input fills" %}
```

Comment:
This is probably a copy/paste error. We don't need to repeat this section. :)

Reply:
D'oh! Good catch. Addressed in https://github.com/StackExchange/Stacks/pull/1646/commits/27f84808d8d9ef27ca7f1ed4f5675a5061c24f55